### DISCUSSION TITLE
printing about the error

### DISCUSSION THREAD

**Discussion by TopedGames** [godotengine/godot-proposals#10371] [Editor]
На компьютере, если при запуске проекта у тебя присутствуют ошибки, то тебя переносит в редактор. На телефоне такого нет, а на телефоне это я считаю крайне полезным и удобным, для быстрого устранения ошибок, без выхода из приложения.

**Comment by AThousandShips** [comment]
The official language of these channels is English, please make any reports or comments in English, if you do not know English please use some translation software or ask for help.

**Comment by TopedGames** [comment]
On a computer, if you have errors when starting a project, then you are transferred to the editor. There is no such thing on the phone, but I find it extremely useful and convenient on the phone, for quick troubleshooting, without exiting the application.

**Comment by AThousandShips** [comment]
This sounds like this issue:
* https://github.com/godotengine/godot/issues/94159

**Comment by TopedGames** [comment]
This does not solve the problem. And it is also extremely inconvenient.

**Reply by AThousandShips** [reply]
Is the issue the same? The workaround isn't important. If this is the same problem this is a duplicate

**Reply by TopedGames** [reply]
Oh, that's it, I just noticed that I need to switch to the application again. Thanks for the help.
